src/Microsoft.Data.SqlClient/tests/tools/Microsoft.DotNet.XUnitExtensions/DiscovererHelpers.cs

```diff
@@ -3,14 +3,15 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Runtime.InteropServices;
```

Comment:
Unnecessary namespace?

Reply:
It's required when you run the tests as "package" reference.